scripts/lib/CIME/XML/archive_base.py

```diff
@@ -99,7 +99,7 @@ def get_all_hist_files(self, casename, model, from_dir, suffix="", ref_case=None
         # remove when component name is changed
         if model == "fv3gfs":
             model = "fv3"
-        if model == "cice6":
+        if model == "cice5":
```

Comment:
the changes in config_files.xml are specific to model cesm.  So the changes in archive_base.py and case_st_archive.py will need to be applied only when cime_model == cesm.

Reply:
Does E3SM have a cice6 or a cice5? Won't it ignore this anyway?

Reply:
Could you just add an .or. statement?  If model = (cice5 or cice6)

Reply:
Actually we don't have cice5 or cice6.  We just have "cice".  So this code probably doesn't affect E3SM.

Reply:
oh good point - it only has cice so I think that it's okay.